pvops/text/preprocess.py

```diff
@@ -6,13 +6,17 @@
 import traceback
 from datetime import datetime, timedelta
 
+from examples.example_data.reference_dict import EQUIPMENT_DICT, PV_TERMS_DICT
+from pvops.text.utils import replace_word_with_pv_term
+
+
 try:
     nltk.data.find('tokenizers/punkt')
 except LookupError:
     nltk.download('punkt')
 
 def preprocessor(
-    om_df, lst_stopwords, col_dict, print_info=False, extract_dates_only=False
+    om_df, lst_stopwords, col_dict, pv_reference_dict=None, print_info=False, extract_dates_only=False
```

Comment:
`text_replace_key_terms` recommended to not be called within preprocessor since not everyone needs to replace key terms and given potential issues with ambiguous labels (eg inv could be inverter or inverse) this step could alter data in a misleading way.

Reply:
this makes sense, moved to utils!